cardy/.babelrc

```diff
@@ -0,0 +1,13 @@
+{
+  "presets": [
+    "env",
+    "react",
+    "stage-0"
+  ],
+  "plugins": [
+    "transform-class-properties",
+    "transform-decorators",
+    "transform-react-constant-elements",
+    "transform-react-inline-elements"
+  ]
+}
```

Comment:
Are all these needed for Fractal?
If so, maybe we could create a Fractal preset and suggest they use it?